### ISSUE TITLE
[2.2] Not loading ./prisma/.env variables when ./.env exists

### ISSUE THREAD

**Issue by jwld** [prisma/prisma#2971]
## Bug description

When running prisma migrate I'm getting the following error:
```
Environment variables loaded from current directory
Error: Get config {"is_panic":false,"message":"error: Environment variable not found: DATABASE_URL.\n  -->  schema.prisma:3\n   | \n 2 |   provider = \"postgresql\"\n 3 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1","meta":{"full_error":"error: Environment variable not found: DATABASE_URL.\n  -->  schema.prisma:3\n   | \n 2 |   provider = \"postgresql\"\n 3 |   url      = env(\"DATABASE_URL\")\n   | \n\nValidation Error Count: 1"},"error_code":"P1012"}
```

Works fine if I store `DATABASE_URL` in a `.env` file at the root level, or cd into prisma before running the commands.

## How to reproduce

Create a schema that uses a `.env` file:
```prisma
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

Store the `.env` file inside the `prisma` folder.

Run `prisma migrate up` or `prisma migrate save`

## Expected behavior

Load the env file from the prisma folder as before.

## Environment & setup

- OS: Mac OS
- Database: PostgreSQL
- Prisma version: 2.2
- Node.js version: 14.2.0

**Comment by janpio** [comment]
So you are indeed correct @jwld - this behavior was different in 2.1.0 and applies to all Prisma CLI commands. I moved this issue to the main repository and asked my colleagues to look into this. Thanks for reporting!

**Comment by janpio** [comment]
Similar report in Community Slack: https://prisma.slack.com/archives/CKQTGR6T0/p1594241396043100?thread_ts=1594236245.032700&cid=CKQTGR6T0

**Comment by Jolg42** [comment]
Thanks for reporting! We have a fix coming up next week 🐛

**Comment by Jolg42** [comment]
Thanks a lot for reporting 🙏

This issue is fixed in the latest version of @prisma/cli: 2.2.1
